files/struct.cpp

```diff
@@ -0,0 +1,22 @@
+#include <iostream>
+
+//wir erstellen ein Struct mit dem Namen Vektor
+struct Vektor{

```

Comment:
wir haben in den früheren Dateien immer ein Leerzeichen vor `{` also `Vecktor {` gilt natürlich auch für `main() {` und so

Reply:
:white_check_mark: